### DISCUSSION TITLE
bigger file size?

### DISCUSSION THREAD

**Discussion by jaymzh** [shaneosullivan/gitmeme#34] [General]
I don't want to blow out whatever @shaneosullivan is paying for the backend service, but do you think we could bump the max filesize up a bit? Like maybe 5MB or something? Most of the animated GIFs from phabricator are well over 1MB.

**Comment by shaneosullivan** [comment]
<a href="https://gitme.me/image?url=https%3A%2F%2Fmedia2.giphy.com%2Fmedia%2FmNwhyN3JyuMykpxaTA%2Fgiphy.gif&token=willdo" data-gitmeme-token="willdo"><img src="https://media2.giphy.com/media/mNwhyN3JyuMykpxaTA/giphy.gif" title="Created by gitme.me with /willdo"
        alt="willdo"/></a>

**Comment by shaneosullivan** [comment]
I increased the file size in https://github.com/shaneosullivan/gitmeme/commit/ad66468153b5a8a46b9b8a1a34642b6f08d6844a , waiting on Google to approve the new version

**Comment by jaymzh** [comment]
Brilliant, thanks!

The meme I want to upload is 4.9MB, so it'll JUST BARELY fit :)

**Comment by jaymzh** [comment]
Your changes to the site don't quite work out though:

![image](https://github.com/shaneosullivan/gitmeme/assets/938467/c755c7e4-46c4-4dd3-ad27-87fb39d86e7c)

Both in FF and Chrome.

**Comment by shaneosullivan** [comment]
Damn, I thought I had that sorted. Will fix

**Comment by jaymzh** [comment]
Uploads of that size seem to just hang for ever. It says `Submitting new image, please wait`, but it just hangs forever.
